ChangeLog.md

```diff
@@ -1,3 +1,15 @@
+# Changes in 1.1.10
```

Comment:
Shouldn't this be 1.1.9? The released version is still 1.1.8.

Reply:
Ah yes possibly